mypy/errors.py

```diff
@@ -618,8 +618,12 @@ def report_internal_error(err: Exception,
 
     # Print "INTERNAL ERROR" message.
     print('{}error: INTERNAL ERROR --'.format(prefix),
-          'please report a bug at https://github.com/python/mypy/issues',
-          'version: {}'.format(mypy_version),
+          'Please try using master\n'
```

Comment:
Nit: it may not be clear what master means in this context. Maybe use 'mypy master on GitHub' or something.